common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4715,6 +4715,14 @@
     ],
     "sqlState" : "42K06"
   },
+  "STATE_STORE_SCHEMA_MUST_BE_NULLABLE" : {
+    "message" : [
+      "If schema evolution is enabled, all the fields in the schema for column family <columnFamilyName> must be nullable.",
+      "Please set the 'spark.sql.streaming.stateStore.encodingFormat' to 'UnsafeRow' or make the schema nullable.",
```

Comment:
This is stored in the offset log though. So maybe just say that they should make schema nullable ?